salesagility/SuiteCRM
# Fixed #2992 - Cannot Duplicate Project Template

## Description



Fixes project templates not being duplicated when a user clicks "Actions -> Duplicate" and instead returning a blank screen.

Issue ref: #2992

## How To Test This

1. Create a project template.
2. Enter project template detail view.
3. Click actions -> duplicate.
4. Check that the template has been duplicated

## Types of changes


### Final checklist
